discord/ext/commands/context.py

```diff
@@ -354,52 +354,47 @@ async def send_help(self, *args: Any) -> Any:
         """
         from .core import Group, Command, wrap_callback
         from .errors import CommandError
+        from .help import _context
 
         bot = self.bot
         cmd = bot.help_command
 
         if cmd is None:
             return None
 
-        cmd = cmd.copy()
-        cmd.context = self  # type: ignore
+        _context.set(self)
+
         if len(args) == 0:
             await cmd.prepare_help_command(self, None)
-            mapping = cmd.get_bot_mapping()
+            entity = cmd.get_bot_mapping()
```

Comment:
This change doesn't seem right.

Reply:
`HelpCommand.send_bot_help` excepts a `Mapping`, so it's correct. 
In an earlier version of this PR i needed to make changes to how the call was done which I no longer do, so technically this refactor of `send_help` could be undone.